system-parachains/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -77,15 +77,17 @@ use frame_system::{
 	limits::{BlockLength, BlockWeights},
 	EnsureRoot,
 };
+// TODO make combined mod to re-export
```

Comment:
Who are these re-exports used by (if anyone)? Is it an artifact from when the runtime code was directly used in `polkadot-parachain` or is this still used in something like RPCs?

There are two options here:
1. Make it clear in each parachain runtime exactly where the types and constants come from (as is done here) but break this API for anybody who uses e.g. `collectives_polkadot_runtime::common::polkadot`
2. Re-export the contents of `parachains-common` in `system-parachains-constants` and keep this `pub use`, but then all of the `use parachains_common::*` statements throughout the parachains will be moved too and it will be less clear whether they are defined in `system-parachains-constants` or `parachains-common`